storage/fifocache/fifocacheSharded.go

```diff
@@ -43,7 +43,7 @@ func (c *FIFOShardedCache) Clear() {
 }
 
 // Put adds a value to the cache.  Returns true if an eviction occurred.
-func (c *FIFOShardedCache) Put(key []byte, value interface{}) (evicted bool) {
+func (c *FIFOShardedCache) Put(key []byte, value interface{}, _ int) (evicted bool) {
```

Comment:
maybe add comment why size is not used here.

Reply:
done